src/Kafka/CreateKafkaInstance/CreateKafkaInstance.stories.tsx

```diff
@@ -386,19 +390,6 @@ VariantOnlySingleAZEnabledWithRightTooltip.args = {
   ),
 } as CreateKafkaInstanceProps;
 
-export const VariantBothAZEnabledWithRightTooltip = Template.bind({});
```

Comment:
Removed this story as it's a duplicate of `VariantBothAvailabilityZonesEnabledWithNoTooltip`